docs/monthly-meeting/2022-09.md

```diff
@@ -0,0 +1,72 @@
+# Documentation Community Team Meeting (September 2022)
+
+- **Date:** 2022-09-12
+- **Time:** [21:00 UTC](https://arewemeetingyet.com/UTC/2022-09-12/21:00/Docs%20Meeting)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- **Discourse thread:** https://discuss.python.org/t/18590
+- **Calendar for future meetings:** (send your e-mail to Mariatta for a calendar invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [hackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+- Petr Viktorin / `@encukou`
+- Ezio Melotti / `@ezio-melotti`
+- Mariatta / `@mariatta`
+- Ned Batchelder / `@nedbat`
+- C.A.M. Gerlach / `@CAM-Gerlach`
+
+
+## Reports and celebrations
+
+- We will be publishing the Workshop videos to YouTube soon. Stay tuned for
+  announcement and emails.
+
+- Ee is back, so several things should get unblocked in the coming weeks,
+  like the YouTube uploads or docs preview on Netlify
+
+- What's New editing (**CAM**)
+  - Major PEP sized items all added except for PEP 670
+  - First set of copyedit PRs waiting on review/merge; please help!
+  - Refs from summary to body content will be PRed very soon
+  - Remaining items will be submitted as PRs soon(™)
+
+- "Past Action Item": Try building automation to prevent link breakage using the .inv file
+  - Moving whole pages could be handled by `sphinxext-rediraffe`,
+    like in the devguide
+  - Moving fragments (the `#` part of URLs, like headings and reference entries)
+    is not handled well
+    - Intersphinx links work well for reference entries, but many external
+      links don't come form Sphinx
+  - Fixing Sphinx roles might break Intersphinx, depending on how the URL
+    is generated
+
+- Fixing bad Sphinx links
+  - Big project, but could be good for newer contributors
+  - **Petr** might have a GH Actions-based per-document check for PRs
+  - Some links, e.g. in old What's New, might need to stay broken.
+    Silencing errors with `!` doesn't always work
+
+- Running linkcheck regularly (on devguide?)
+  - Seems like a good idea, just needs someone to implement
+    (**Ezio**, or **CAM** if needed)
```

Comment:
```suggestion
  - Seems like a good idea, just needs someone to implement
    (**Ezio**, or **CAM** if needed)
  - https://github.com/python/devguide/issues/681
  - https://github.com/python/devguide/issues/917
```

Reply:
```suggestion
  - Seems like a good idea, just needs someone to implement
    (**Ezio**, or **CAM** if needed)
  - See [python/devguide#681](https://github.com/python/devguide/issues/681) and [python/devguide#917](https://github.com/python/devguide/issues/917)
```

Again, may as well use proper links here—more compact, easier to read and cleaner/less noisy